.readthedocs.yml

```diff
@@ -5,8 +5,13 @@ build:
   tools:
     python: "mambaforge-4.10"
 
+  # Append fixed Python version to requirements
+  jobs:
+    pre_create_environment:
+      - echo "  - python=3.9" >> requirements/env_climada.yml
```

Comment:
Great! We just need to remember to update this line when we update the Python version.

Reply:
The "good" thing is that the RTD build will break once we update the `python_requires` in `setup.py` without updating this line here. So, we will be reminded anyway 🙂